airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1146,14 +1146,19 @@ components:
           type: string
         conn_type:
           type: string
+          nullable: true
```

Comment:
This fielld should be non-nullable.  I created separate PR about the database schema.
https://github.com/apache/airflow/pull/9187

Reply:
Ok. Can you do rebase? My change has been merged.

Reply:
I hurriedly pushed now. I will still come back to it this night. [e40d532](https://github.com/apache/airflow/pull/9095/commits/e40d532c01461672514fa07291633bb3796fd9b6)